File: questao4.py
""" Escreva uma função que receba dois números positivos por parãmetro e retorne a soma dos N números
inteiros existentes entre eles.
"""

def somaInt(num1, num2):
    for numero in range(num1, num2 + 1, 1):
        if numero == num1: 
            soma = numero
        else: 
            soma += numero
            print(f' somando {numero} fica {soma}')

    return(soma)

n1 = int(input("Digite o primeiro número: "))
n2 = int(input("Digite o segundo número: 5"))
print(f' A soma dos números inteiros entre {n1} e {n2} = {somaInt(n1, n2)}')

File: questao4.c
/*  Escreva uma função que receba dois números positivos por parãmetro e retorne a soma dos N números
inteiros existentes entre eles.   */
/*OBS: Demorei bastante nessa tentando lebrar da sintaxe da
função*/
#include <stdio.h>



int somaInt(int num1, int num2){
    int soma = 0;
    
    for(int i= num1; i<= num2; i++){
        if(i == num1){
            soma = num1;
        }
        else{
            soma += i;
        }
        printf("Adicionando o numero [%i] a soma seria [%i]\n", i, soma);
    }
    
    
    return(soma);
}

int main(){
    int num1, num2, variavel;
    
    printf("Digite o primeiro numero: ");
    scanf("%i", &num1);

    printf("Digite o segundo numero: ");
    scanf("%i", &num2);

    printf(" A soma dos numeros entre [%i] e [%i] sera [%i]\n", num1, num2, somaInt(num1, num2));

    return 0;
}